mreg_cli/policy.py

```diff
@@ -522,7 +522,7 @@ def host_add(args):
         path = f'/api/v1/hostpolicy/roles/{args.role}/hosts/'
         history.record_post(path, "", data, undoable=False)
         post(path, **data)
-        cli_info(f"Added {name} to {args.role}", print_msg=True)
+        cli_info(f"Added host {name} to role {args.role}", print_msg=True)
```

Comment:
Should we quote the variables to make them stand out a bit better? Ala:
`cli_info(f"Added host '{name}' to role '{args.role}'", print_msg=True)`

Reply:
Sure, why not. Fixed in [99d4484](https://github.com/unioslo/mreg-cli/pull/164/commits/99d4484d693a2d4ec55cc909a7ac22da650cec23)